htdocs/ticket/class/ticket.class.php

```diff
@@ -381,6 +381,7 @@ private function verify()
 
 		if (isset($this->message)) {
 			$this->message = trim($this->message);
+			if (dol_strlen($this->message) > 65000) $this->message = dol_substr($this->message, 0, 65000, 'UTF-8') . '...';
```

Comment:
Truncating may generate not valid html. Why doing
```
$this->errors[] = 'ErrorFieldTooLarge';
dol_syslog(get_class($this)."::create error -1 ref null", LOG_ERR);
$result = -1;
```

Reply:
I will also change size of field into mediumtext instead of text

Reply:
@eldy 
I did what you suggested to me.
The translate key "ErrorFieldTooLarge" doesn't exist
So, I replaced "ErrorFieldTooLarge" with "ErrorFieldTooLong".
And the mediumtext size had already been changed in your commit : https://github.com/Dolibarr/dolibarr/commit/60133805ee1b2a8b27d9a3b75ccaa2ed901e08bb